### PR TITLE
3153 var missing in plot proc.py

### PR DESCRIPTION
**Pull request by chris-ashe**
## Description
There has been several problems in `plot_proc.py` that prevent it from showing all of the variables correctly and formatted neatly

**Changes include:**

- The dark blue WP is now **no longer plotted** in the toroidal cross-section plot as this would normally overlap with the TF coil case lines. The TF plotter on page 3 now does this better.
image
-  The "plasma" and "cryostat" labels have been removed from the toroidal cross-section plot to prevent cluttering.

- The vertical thermal shield (`thsield_vb`) was not importable as it was not outputted to the MFILE, this is now resolved.
- The width of the neutral beam duct (`beamwd`) was being outputted to the MFILE with its default value when a design did not use NBI, this has now been removed from the output.
- The spacing between the label names and colours for the cross section plots have been spaced out to prevent overlap of text.
- Variable descriptions all now have correct capitalisation.
- Change of list indexing now sees `plot_proc.py` know the correct number of PF coils. Preventing `rpf[n]` not found errors.
-  The colour legend is now moved to the top right of the first page to prevent clashing with the plasma composition output 
image
- The inclusion of fixed secondary heating and current drive is now properly implemented preventing NaN errors and saying there is secondary heating when it is not present.
- `wwp2` is now only imported when the WP design that uses it (`i_tf_wp_geom == 1`) is used. It was previously wrongly imported when  `i_tf_turns_integer == 0`.
- `nbshield, rtanbeam, rtanmax` and `beamwd` are now only imported when NBI is used, as they are not in the MFILE otherwise.
- Scaling law name output is now back thanks to #3156 
- `i_tf_wp_geom` and `i_tf_case_geom` output has now been fixed to show integers as they are switch variables.
- `kallenbach_switch` has been removed as the Kallenbach models are no longer used.
- Outputs using `alstrtf` have been removed as the variable is obsolete.
- Spacing between all output labels and values has been increased to prevent overlap.

- The numbering of the PF coils has been better adjusted to fit properly within the boxes without colliding with the box edges
- "Th shield" is now "Thermal shield" in the plotting key.
- NB duct is now only shown in the colour key when NBI is present
- A new colour scheme has been added for the cross sections to match that used in [BLUEMIRA](https://github.com/Fusion-Power-Plant-Framework/bluemira), it can be set in the CLI with `--colour`
image

- The plasma colour fill now completely fills the plasma boundary with no over spill

image

## Checklist

I confirm that I have completed the following checks:

- [x] I have justified any large differences in the regression tests caused by this pull request in the comments.
- [x] I have added new tests where appropriate for the changes I have made.
- [x] If I have had to change any existing unit or integration tests, I have justified this change in the pull request comments.
- [x] If I have made documentation changes, I have checked they render correctly.
- [x] I have added documentation for my change, if appropriate.

### CODE REVIEW

**Review by j-a-foster — APPROVED** [review]
Tested the new plot_proc functionality and checked that issues described in PR description have been addressed. Ready for merge.

**Review by j-a-foster — APPROVED** [review]
Happy with the changes to the docs. Ready for merge.

File: documentation/proc-pages/io/utilities.md [hunk 1]
```diff
@@ -105,71 +108,88 @@ Produces a three-page PDF file in the same directory as the input MFILE. The PDF
...
+| The steady state auxiliary power used for heating and current drive during the flat top phase (NOT to be confused with the start up or ramp down power requirements) |
+| Part of the auxiliary power that is used for heating only, but not current drive                                                                                     |
+| Current drive fractions for the bootstrap auxiliary and inductive current                                                                                           |
+| The neutral beam current drive efficiency, $\gamma_{NB}$ (If NBI used)                                                                                                             |
+| The neutral beam energy (If NBI used)                                                                                                                                             |
+| The plasma heating used in the calculation of the confinement scaling / H-factor, $P_{\text{aux}} + P_\alpha - P_{\text{rad,core}}$                                                   |
+|The normalised current drive efficiency, [$\text{A/W}\text{m}^2$]|
```
[the remark this answers is not in the document]
Comment (chris-ashe) [reply documentation/proc-pages/io/utilities.md]: Unit removed

File: documentation/proc-pages/io/utilities.md [hunk 2]
```diff
@@ -105,71 +108,88 @@ Produces a three-page PDF file in the same directory as the input MFILE. The PDF
...
-| definitions of $\beta$ as given in [^1]                                                                         |
-| volume averaged electron temperature $\langle T_e\rangle$ and density $\langle n_e\rangle$                      |
-| fraction of the line averaged electron density over the Greenwald density $\langle n_{e,line}\rangle / n_{GW}$  |
-| peaking of the electron temperature $T_{e,0}/\langle T_e\rangle$ and density $n_{e,0}/\langle n_{e,vol}\rangle$ |
-| core and SoL effective charge $Z_{eff}=\sum_i f_iZ_i^2$                                                         |
-| impurity fraction $f_Z=n_Z/\langle n_e\rangle$                                                                  |
-| H-factor and confinement time are calculated using a radiation corrected confinement scaling[^3] [^4].          |
-
-| Neutral Beam Current Drive                                                                                                                                           |
+| Plasma current, $I_{\text{P}}[\text{MA}]$                                                                                        |
```
[the remark this answers is not in the document]
Comment (chris-ashe) [reply documentation/proc-pages/io/utilities.md]: Unit removed

File: documentation/proc-pages/io/utilities.md [hunk 3]
```diff
@@ -105,71 +108,88 @@ Produces a three-page PDF file in the same directory as the input MFILE. The PDF
...
-| L-H threshold power $P_{LH}$                                                                                |
-| divertor lifetime in years                                                                                  |
-| high grade heat for electricity production $P_{therm}$                                                      |
-| gross cycle efficiency $P_{e,gross}/P_{therm}$                                                              |
-| net cycle efficiency $\frac{P_{e,gross}-P_{heat,pump}}{P_{therm}-P_{heat,pump}}$                            |
-| net electric power $P_{e,net}=P_{e,gross}-P_{recirc}$                                                       |
-| plant efficiency $P_{e,net}/P_{fus}$                                                                        |
+| Nominal neutron wall load[^2]                     |
+| Normalised radius of the 'core' region $\rho_{core}$ used in the radiation correction of the onfinement scaling[^3] [^4]               |
+| The electron density at the pedestal top, $n_{\text{e,ped}}[\text{m}^{-3}]$                                               |
```
[the remark this answers is not in the document]
Comment (chris-ashe) [reply documentation/proc-pages/io/utilities.md]: Unit removed

File: documentation/proc-pages/io/utilities.md [hunk 4]
```diff
@@ -105,71 +108,88 @@ Produces a three-page PDF file in the same directory as the input MFILE. The PDF
...
+| Aspect ratio, $A$                                           |
+| Elongation at the 95% flux surface, $\kappa_{95}$          |
+| Plasma triangularity at the 95% flux surface, $\delta_{95}$|
+| Plasma surface area                                        |
+| Plasma volume                                              |
+| Number of TF coils                                         |
+| Inboard blanket + shield                                   |
+| Outboard blanket + shield                                  |
+| Total fusion power                                         |
+| $Q_{\text{p}}$                                             |
```
[the remark this answers is not in the document]
Comment (chris-ashe) [reply documentation/proc-pages/io/utilities.md]: Description added

File: documentation/proc-pages/io/utilities.md [hunk 5]
```diff
@@ -105,71 +108,88 @@ Produces a three-page PDF file in the same directory as the input MFILE. The PDF
 
 `Plasma Composition` - Number densities of several ion species relative to the electron density.
 
-`Coil Currents etc` - Peak coil currents of the PF coils in $MA$, flux swing of the central solenoid 
-used for startup and total available in $Wb$. Total burn time `tburn` in hrs.
+`Coil Currents etc` - Peak coil currents of the PF coils in $\text{MA}$, flux swing of the central solenoid 
+used for startup and total available in $\text{Wb}$. Total burn time `tburn` in hrs.
 
-`Cost of electricity` - This is the cost of electricity in $/MWh$. Check the respective cost model 
+`Cost of electricity` - This is the cost of electricity in $/\text{MWh}$. Check the respective cost model
```
[the remark this answers is not in the document]
Comment (chris-ashe) [reply documentation/proc-pages/io/utilities.md]: Added in $ symbol

**Review by j-a-foster — APPROVED** [review]
Happy with changes to docs. Ready for merge.

**Pull request merged**